uenyingwa/blogga
# #148189043 initial project setup

#### What does this PR do?
This PR setups the rails project.

#### Description of Task to be completed?
AC:
* As a user I should be able to see an initial rendering on the homepage.

#### How should this be manually tested?
- Start the server
- Open any web browser and enter the address `http://localhost:3000/articles` to see the list of articles. 

#### What are the relevant pivotal tracker stories?
#148189043

#### Screenshots (if appropriate)
<img width="1663" alt="screen shot 2017-07-01 at 9 34 49 pm" src="https://user-images.githubusercontent.com/24937935/27765291-b6c5ec04-5ea5-11e7-9992-5cbda8d0bbe5.png">
